crates/vm/levm/src/constants.rs

```diff
@@ -7,7 +7,7 @@ pub const SUCCESS_FOR_CALL: i32 = 1;
 pub const REVERT_FOR_CALL: i32 = 0;
 pub const HALT_FOR_CALL: i32 = 2;
 pub const SUCCESS_FOR_RETURN: i32 = 1;
-pub const REVERT_FOR_CREATE: i32 = 0;
+pub const CREATE_DEPLOYMENT_FAIL: i32 = 0;
```

Comment:
Couldn't this constant be a U256 directly? It seems like that we are casting it every time we use it.

What do you think?

```suggestion
pub const CREATE_DEPLOYMENT_FAIL: U256 = U256::zero();
```

Reply:
True, I will update you when I change it.

Reply:
https://github.com/lambdaclass/ethrex/pull/1460/commits/91a031f4796d40d136605d0dd2d9cf51f7ff920c

Reply:
I did it in these commits.
https://github.com/lambdaclass/ethrex/pull/1460/commits/a7a0cf55ea35a4972ea970360f8f12b93e58f6e9
https://github.com/lambdaclass/ethrex/pull/1460/commits/91a031f4796d40d136605d0dd2d9cf51f7ff920c
Sorry that I divided the changes in 2 and mixed them up with other things